src/simulator/cube.py

```diff
@@ -11,29 +10,38 @@ def __init__(self, length: float, height: float, width: float):
         self.height = height
         self.width = width
 
-    def bounding_box(self) -> Box:
+    def bounding_box(self, position: Position) -> Box:
         """Returns smallest enclosing upright Box
-        """
-
-        # TODO: The following calculation is only for an upright cube
-        # dimension calculations
-        length = self.shape.length
-        height = self.shape.height
-        width = self.shape.width
 
-        # position calculations
-        x = self.position.x
-        y = self.position.y
-        z = self.position.z
-        phi = self.position.phi
-        theta = self.position.theta
+           Calculation the smallest bounding box,function moves
```

Comment:
nice explanation. Thanks. Some spelling and grammaer mistakes. (same for Cylinder)